src/modules/blog-category/blog-categories.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { BlogCategory } from './entities/blog-category.entity';
+import { BlogCategoriesService } from './blog-categories.service';
+import { SuperAdminGuard } from 'src/guards/super-admin.guard';
```

Comment:
The path used will fail if there is a change.

```suggestion
import { SuperAdminGuard } from '../../guards/super-admin.guard';
```